modules/layers/src/text-layer/multi-icon-layer/multi-icon-layer.js

```diff
@@ -21,8 +21,12 @@
 import IconLayer from '../../icon-layer/icon-layer';
 
 import vs from './multi-icon-layer-vertex.glsl';
+import fs from './multi-icon-layer-fragment.glsl';
 
 const defaultProps = {
+  fontSmoothing: 0,
+  cutoff: 192.0 / 255.0,
```

Comment:
- Define [prop type](http://deck.gl/#/documentation/developer-guide/writing-custom-layers/layer-prop-types) and range
- Add a comment on where this number comes from